hoomd/mpcd/CollisionMethod.h

```diff
@@ -113,6 +119,7 @@ class PYBIND11_EXPORT CollisionMethod : public Autotuned
         }
 
     protected:
+    std::shared_ptr<mpcd::CellThermoCompute> m_thermo;         //!< Cell thermo
```

Comment:
Give this some other name, and change above. Also make sure to call its `compute()` method right before you do the momentum transfer.
```suggestion
    std::shared_ptr<mpcd::CellThermoCompute> m_thermo_rigid_thermostat;         //!< Cell thermo
```